.github/workflows/ISSUE_CREATION_WORKFLOW.yml

```
@@ -4,83 +4,70 @@
 # documentação.
 
 name: Weekly Team Sync
+
 on:
   workflow_dispatch:
     inputs:
       input:
         description: "Criação de issue"
         required: false
         default: "Issue criada"
-    
+
   schedule:
     - cron: 0 12 * * 4
 
 env:
-    MONTH_01: "Janeiro"
-    MONTH_02: "Fevereiro"
-    MONTH_03: "Março"
-    MONTH_04: "Abril"
-    MONTH_05: "Maio"
-    MONTH_06: "Junho"
-    MONTH_07: "Julho"
-    MONTH_08: "Agosto"
-    MONTH_09: "Setembro"
-    MONTH_10: "Outubro"
-    MONTH_11: "Novembro"
-    MONTH_12: "Dezembro"
+  MONTH_01: "Janeiro"
+  MONTH_02: "Fevereiro"
+  MONTH_03: "Março"
+  MONTH_04: "Abril"
+  MONTH_05: "Maio"
+  MONTH_06: "Junho"
+  MONTH_07: "Julho"
+  MONTH_08: "Agosto"
+  MONTH_09: "Setembro"
+  MONTH_10: "Outubro"
+  MONTH_11: "Novembro"
+  MONTH_12: "Dezembro"
+  # Change this variable for changing the meeting weekday
+  DATE_WEEKDAY: "wednesday"
+  GRUPO_DE_TRABALHO_OU_TEMA_DA_REUNIAO: "Geral"
 
 jobs:
   create_issue:
     name: Create team sync issue
     runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     permissions:
       issues: write
     steps:
-      - name: Set current day
-        id: day
-        run: echo "::set-output name=day::$(date -d 'next wednesday' +%d)"
+      - name: Extract issue template and generate title
+        uses: imjohnbo/extract-issue-template-fields@v0.0.1
+        id: extract
+        with:
+          path: .github/ISSUE_TEMPLATE/reuniao.md
 
-      - name: Set current month
-        id: month
-        run: echo "::set-output name=month::$(date -d 'next wednesday' +%m)"
+      - name: Update issue template title
+        id: title
+        run: |
+          day=$(date -d "next $DATE_WEEKDAY" +%d)
+          month_num=$(date -d "next $DATE_WEEKDAY" +%m)
+          month_var="MONTH_$month_num"
+          month=$(eval echo ${!month_var})
+          year=$(date -d "next $DATE_WEEKDAY" +%Y)
 
-      - name: Set final month name
-        id: month_name
-        run: echo "::set-output name=month_name::${!MONTH_NAME_VAR}"
-        env:
-          MONTH_NAME_VAR: MONTH_${{ steps.month.outputs.month }}
+          DATA_DA_REUNIAO="$day de $month de $year"
+          title=$(eval echo ${{ steps.extract.outputs.title }})
```

Acho que entendi, então no caso seria apenas mudar para isso?
```suggestion
          title="[Reunião] $GRUPO_DE_TRABALHO_OU_TEMA_DA_REUNIAO: $DATA_DA_REUNIAO"
```

O segredo da no eval né? Se for só isso mesmo, talvez seja melhor manter o título como antes, mas qualquer coisa além disso não precisa mexer.

---

Exatamente! Vou manter o título como estava então.